frontend/src/__tests__/integration/pages/modelServing/ModelServingGlobal.spec.ts

```diff
@@ -1,10 +1,36 @@
 import { test, expect } from '@playwright/test';
+import { navigateToStory } from '~/__tests__/integration/utils';
 
-test('Delete model', async ({ page }) => {
+test('Empty State No Serving Runtime', async ({ page }) => {
   await page.goto(
-    './iframe.html?args=&id=tests-integration-pages-modelserving-modelservingglobal--delete-model&viewMode=story',
+    navigateToStory('modelserving-modelservingglobal', 'empty-state-no-serving-runtime'),
```

Comment:
I've just created this to open a conversation, I think @andrewballantyne mentioned this a few weeks ago, but I couldn't find any logged issue, should we include this util method to keep the navigation more consistent?